src/FischBot/Services/HalfMastService/HalfMastService.cs

```diff
@@ -16,7 +17,7 @@ public class HalfMastService : IHalfMastService
         private readonly string _twitterApiAccessToken;
         private readonly string _twitterApiAccessTokenSecret;
 
-        private const string _halfMastAlertsTwitterAccountName = "HalfStaffAlerts";
+        private const string HalfMastAlertsTwitterAccountName = "HalfStaffAlerts";
```

Comment:
this is a private field, and matching the style of the rest of the project should be lower camelcased, prefaced by an underscore, as it was before.

Please change this back to `_halfMastAlertsTwitterAccountName`